### ISSUE TITLE
Snakemake and cmake mentioning in make slides?

### ISSUE THREAD

**Issue by jpherdi** [pep-dortmund/toolbox-workshop#300]
What is your opinion?

**Comment by chrbeckm** [comment]
Too advanced for most of the participants, but nice addition to https://toolbox.pep-dortmund.org/links.html#make

**Comment by maxnoe** [comment]
Cmake doesn't really fit here. Snakemake is a good more modern, pythonic general tool but cmake is very specific to building c++ projects and not a replacement at all for make as we use it in the Workshop.

**Comment by jpherdi** [comment]
Yes, I like snakemake quite a lot as a replacement for make, especially because it is pythonic which fits well in the toolbox workshop.

Yes, cmake I would only mention. Could be nice to use in bigger projects in Computational Physics maybe, to get an idea. But nevermind.

Maybe in the future we could think about replacing make with snakemake in the workshop
